etc/profile-a-l/sway.profile

```diff
@@ -0,0 +1,17 @@
+# Firejail profile for Sway
+# Description: i3-compatible Wayland compositor 
+# This file is overwritten after every install/update
+# Persistent local customizations
+include sway.local
+# Persistent global definitions
+include globals.local
+
+# all applications started in sway will run in this profile
+noblacklist ${HOME}/.config/sway
```

Comment:
Add a `blacklist` for it in disable-common.inc.

IIRC sway uses `~/.config/i3` as fallback if there is no `~/.config/sway`. We should add `nobalcklist ${HOME}/.config/i3`.

Reply:
done